_extras/partner_policy.md

```diff
@@ -0,0 +1,41 @@
+* The number of seats your organization receives depends on your membership level (Silver/Gold/Platinum).  At least six participants must attend at once.  Any unused seats will be forfeited. This is to ensure we can accommodate everyone across all our partner sites. Our  trainings include group exercises that  your  team must work on together.  We also want our training cohorts to recognize they are part of a team which represents the Carpentries at their institution.
+* Again, because we have group exercises, the entire team must be in the same room.  Individuals can not participate remotely.  
+* If an individual misses more than one hour of the training over both days, he/she will not pass the training requirement and his/her spot will be forfeited.  We are training people to deliver Carpentries training and be ambassadors of the Carpentries program, and need to ensure they get our full experience.
+* Names and emails of participants must be shared with Carpentries staff at least three weeks before the event.  Participants will be asked to complete our application form to register in our system (https://amy.software-carpentry.org/forms/request_training/).  Participants will receive a link to more information including some pre-reading materials in advance of the event.  
+* Exceptions to any of the above may be granted at the discretion of Carpentries staff on a case by case basis. For questions please contact maneesha@carpentries.org.
+
+
+Once you’ve confirmed dates and participants, here is some information about preparing for the event itself:
+
+* Trainings are offered remotely.  Your room should be set up for videoconferencing with a large screen & projector, speakers, camera, and microphone.
+* The training room should have wifi capable of videoconferencing and for each participant laptop to connect.
+* Participants should bring their own laptop - not a Chromebook, tablet, etc.  No special software other than a modern internet browser is required.
+* Schedule will include a lunch break as well as mini breaks during the day.  We strongly encourage each site to provide coffee and lunch to help build team cohesiveness and to maintain the schedule.
+* Carpentries will provide a link to host the videoconference via Blue Jeans in advance of the event in order to test your setup.
+
+
+Lastly, some information about what happens after the event:
+
+* Participants will have three months (90 days) to complete post-training homework exercises required to receive their instructor certificate (http://swcarpentry.github.io/instructor-training/checkout/).   
+* Participants who fail to complete these exercises will not be eligible for instructor certification. Extensions may be granted at the discretion of Carpentries staff. 
```

Comment:
Extensions may be granted at the discretion of Carpentries staff, but these must be applied for within the 90 days limit.